client/model.go

```diff
@@ -286,6 +286,7 @@ type Environment struct {
 	LifespanEndAt               string        `json:"lifespanEndAt"`
 	LatestDeploymentLogId       string        `json:"latestDeploymentLogId"`
 	LatestDeploymentLog         DeploymentLog `json:"latestDeploymentLog"`
+	IsArchived                  bool          `json:"isArchived"`
```

Comment:
don't we need to use it on the logic somewhere?
just wondering

Reply:
The PR handled also environment resource at start but since it isn't so trivial to re-create an environment on some drift and there's a deeper product decision to make before implementing it, we've decided to ditch it for now and create a separate issue for environment and project resources.
This looks like a leftover tho.

@samuel-br did you create that seperate issue btw?

Reply:
@yaronya do you mean for https://github.com/env0/terraform-provider-env0/issues/225?

Reply:
I'm not so sure. I expect to see an issue that includes environment and project resources in its description, but couldn't find it.

If I get that correctly:
 - #205 talks about our drift issues in general
 - #225 tries to tackle specific resources, such as template for e.g
 - We're missing another issue that tackles environment and project

Reply:
nope, Ill do it later today, but I want  to summary this before on channel, cause for now there is three or for types of resources in this contexts of problem

Reply:
I added #250 now.
In conclusion we have the main issue #205 that blocked for now because lack of info in returned errors
#225 take care the resources that have `IsDeleted ` attribute except project and environment, those mentioned in #250

Reply:
@samuel-br maybe just change their title?
Handle drift - general
Handle drift - specific resources
etc

and link them

Reply:
i am confused with all of those issues.
I just asked to make the PR with a good description (copy-paste the issue if needed to the PR).

anyways, if that line of code is related to this PR keep it.
if not - remove it

@samuel-br 
cc @eranelbaz

Reply:
I've changed the title for #250 already FYI.
And yes, prefer not having multiple issues with the same name.

Reply:
@samuel-br can you please share on the main issue (i.e #205) what's needed to be done basically and why it differs from the other issues (parsing the error codes from http client layer and etc.)

Reply:
@yaronya @eranelbaz  I changed the issues names (@yaronya thanks on #250 ) and add https://github.com/env0/terraform-provider-env0/issues/205#issuecomment-1062822911 on the main issue, please take a look on that comment to ensure its clear enough